tools/ddsrouter_yaml_validator/ddsrouter_yaml_validator/ddsrouter_config_schema.json

```diff
@@ -35,7 +35,7 @@
                 },
                 "participants":{
                     "type":"array",
-                    "minItems":2,
+                    "minItems":1,
```

Comment:
This must be agreed by all, and in positive case, documentation should be added.

Reply:
Good for me, @rsanchez15?